### ISSUE TITLE
UI: Mind acronyms

### ISSUE THREAD

**Issue by machawk1** [N0taN3rd/wail#51]
"URL" is an acronym. Use this capitalization, not "Url". Apply this elsewhere.

![screen shot 2016-08-30 at 5 13 31 pm](https://cloud.githubusercontent.com/assets/2514780/18107227/1f5663f0-6ed5-11e6-901e-62bb77b1d392.png)

**Comment by machawk1** [comment]
Most of these appear to be fixed, @N0taN3rd , except for the far left header under the Heritrix tab. Change this one then this ticket can be closed.

**Comment by machawk1** [comment]
@N0taN3rd Despite #61 being closed, this issue remains in 1.1.0beta c3c9553c9fd28b75e2fb4b14182c2f1e2d9fc82f
![screen shot 2017-04-17 at 2 39 21 pm](https://cloud.githubusercontent.com/assets/2514780/25099849/ee846996-237b-11e7-8b4e-4ade7b072e4a.png)

**Comment by machawk1** [comment]
Another example, when adding seeds from the file system. All letters in (W)ARC should be capitalized. Version 1.1.0beta on macOS 10.12.4

![screen shot 2017-05-10 at 2 32 00 pm](https://cloud.githubusercontent.com/assets/2514780/25914794/73ee22a2-358d-11e7-9185-1b93a0a5b059.png)

**Comment by N0taN3rd** [comment]
@machawk1 closed as of a3936ad  :feelsgood:
[acronyms.js](https://github.com/N0taN3rd/wail/blob/wail-electron/wail-ui/constants/acronyms.js)
```js
export default {
  url: 'URL',
  urls: 'URLs',
  oneOrMoreUrls: 'URL(s)',
  urir: 'URI-R',
  urim: 'URI-M',
  ui: 'UI',
  warc: 'WARC',
  warcOrArc: '(W)ARC',
  warcOrArcs: '(W)ARCs',
  fileName: 'Filename',
  wail: 'WAIL',
  wailWarcreate: 'WAIL-WARCreate',
  jdk: 'JDK',
  macOs: 'MacOS'
}
```
used by [uiStrings.js](https://github.com/N0taN3rd/wail/blob/wail-electron/wail-ui/constants/uiStrings.js)
40 files refactored to use uiStrings.js
